transcom/mymove
# MB-6067 TIO payment requests all rejected service items show in queue

## Description

> As a TIO I want to be able to see payment requests that have no approved service items, so that I can get a complete picture of what has been requested by the Prime.
> 
> Given that I am a TIO
>  When I filter my queue for 'Reviewed' payment requests
>  Then I see Payment Requests that have been fully rejected in addition
>       to those that have a mix of accepted/rejected service items

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

**TEST**
1. Log in as TIO
2. Click on status filter
3. Click on `Rejected`
4. See rejected payment service items in queue

## Code Review Verification Steps


 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-6067) for this change

## Screenshots


https://user-images.githubusercontent.com/1522549/104529286-3d911e80-55be-11eb-9e95-01783ab344f3.mov
